src/utils/TimeSlots.js

```diff
@@ -43,6 +43,19 @@ export function getSlotMetrics({ min: start, max: end, step, timeslots }) {
       )
     }
   }
+  
+  const lastSlotMinFromStart = slots.length * step
```

Comment:
lets add a comment as to why this needs to be added? and also check on DST that it didn't break that day. 

A test as well would be appreciated :)

Reply:
There wasn't any tests for this util yet, so I wasn't sure how extensive you wanted me to be. Hope the tests I pushed will suffice!